ops/common-args.libsonnet

```
@@ -0,0 +1,18 @@
+local {mixinRolloutNodes, mixinAllChains, ...} = import '../util/mixin.libsonnet';
+
+{
+	mixinExtraArgsAllNodes(chain, commonArgs):
+	chain + mixinRolloutNodes(chain,
+		function(node) node { extraArgs: commonArgs },
```

```suggestion
		function(node) node { extraArgs+: commonArgs },
```

Чтобы не затереть, а дополнить аргументы